### PR TITLE
AEA-365: Add FHIR validator as a submodule.

### PR DESCRIPTION
**Pull request by mfjarvis**

### CODE REVIEW

File: azure/azure-build-pipeline.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ resources:
       type: github
       name: NHSDigital/api-management-utils
       endpoint: NHSDigital
+      ref: AEA-365-checkout-submodules
```
Comment (mfjarvis) [inline azure/azure-build-pipeline.yml:21]: Will need reverting after the utils changes are merged in

**Review by booshibooshi — APPROVED** [review]

**Review by conner-jowitt — APPROVED** [review]

**Pull request merged**